FoodSuggestions/app/src/main/java/com/example/foodsuggestions/main/ContActivity.java

```diff
@@ -0,0 +1,43 @@
+package com.example.foodsuggestions.main;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.MenuItem;
+
+import com.example.foodsuggestions.R;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
+public class ContActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_cont);
+
+        BottomNavigationView bottomNavigationView = findViewById(R.id.idNavbar);
+        bottomNavigationView.setSelectedItemId(R.id.nav_account);
+        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                switch (item.getItemId()){
+                    case R.id.nav_home:
+                        startActivity(new Intent(ContActivity.this, HomeActivity.class));
+                        overridePendingTransition(0,0);
```

Comment:
Please investigate to see if it's needed.

Reply:
Done